README.md

```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 
+
+## Install
+
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
+```
+git clone https://github.com/mozilla/DeepSpeech
+cd DeepSpeech
+git lfs fetch
+git lfs checkout
+sudo pip install -r doc/requirements.txt
```

Comment:
We shouldn't require root access, people may need to work on a box where they can't sudo.

Also, and this is my fault, the kenlm install `https://github.com/kpu/kenlm/archive/master.zip` is missing from `doc/requirements.txt`. Could you add it there?